src/Domain.LinnApps/Requisitions/CreationStrategies/LdReqWithNominalCreationStrategy.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace Linn.Stores2.Domain.LinnApps.Requisitions.CreationStrategies
+{
+    using Linn.Common.Authorisation;
+    using Linn.Common.Logging;
+    using Linn.Common.Persistence;
+    using Linn.Stores2.Domain.LinnApps.Accounts;
+    using Linn.Stores2.Domain.LinnApps.Exceptions;
+    using Linn.Stores2.Domain.LinnApps.Parts;
+    using Linn.Stores2.Domain.LinnApps.Stock;
+    using System.Threading.Tasks;
+
+    public class LdReqWithNominalCreationStrategy : LdreqCreationStrategy, ICreationStrategy {
+        public LdReqWithNominalCreationStrategy(IAuthorisationService authService,
+            IRepository<RequisitionHeader, int> repository, IRequisitionManager requisitionManager, ILog logger,
+            IRepository<Employee, int> employeeRepository, IRepository<Part, string> partRepository,
+            IRepository<StorageLocation, int> storageLocationRepository,
+            IRepository<Department, string> departmentRepository,
+            IRepository<Nominal, string> nominalRepository) : base(authService, repository, requisitionManager, logger,
```

Comment:
not sure if just github or  if these are all one one mega line - should ideally be one param per line when lots of params. Although if you refactor to inject an LdreqCreationStrategy then you'll only have one param to worry about